client/state/data-layer/wpcom/comments/index.js

```diff
@@ -20,6 +21,16 @@ import { errorNotice, successNotice } from 'state/notices/actions';
 import { getSitePost } from 'state/posts/selectors';
 import { getPostOldestCommentDate, getPostNewestCommentDate } from 'state/comments/selectors';
 import getSiteComment from 'state/selectors/get-site-comment';
+import { decodeEntities } from 'lib/formatting';
+
+const commentsFromApi = comments =>
+	comments.map( comment => ( {
+		...comment,
+		author: {
```

Comment:
what happens if the source author is null or missing entirely?

Reply:
i'll add some guards!